src/epitech_calendar.py

```diff
@@ -3,10 +3,13 @@
 import requests
 from datetime import datetime, timedelta
 
+headers = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3'
+}
 
 def get_epitech_login(epitechCookie):
     url = 'https://intra.epitech.eu/user/?format=json'
-    user_data = requests.get(url, cookies={'user': epitechCookie}).json()
+    user_data = requests.get(url, cookies={'user': epitechCookie}, headers=headers).json()
```

Comment:
Has this been a source of error/crash for you?

Reply:
Yes, that's why I corrected it. Also, my requests weren't being sent and I had to add headers to the get